src/ChainSafe.Gaming.InProcessSigner/AccountProvider.cs

```diff
@@ -0,0 +1,24 @@
+using Nethereum.Web3.Accounts;
+
+namespace ChainSafe.Gaming.InProcessSigner
+{
+    /// <summary>
+    /// Provides the current connected account.
+    /// </summary>
+    public class AccountProvider
```

Comment:
Why isn't this an interface that can be injected as DI?

Reply:
It's already injected as an instance since this is the only implementation we have/ever need, if we got the interface route it's just adding more classes imo